backend/src/vehicles/vehicles.controller.ts

```diff
@@ -32,9 +33,12 @@ export class VehiclesController {
   })
   @Get()
   async findAll(): Promise<VehicleGetAllResponseDto> {
-    const vehicles = await this.vehicleService.findAll();
-
-    return { vehicles };
+    // eslint-disable-next-line no-useless-catch
```

Comment:
This rule here is absolutely correct. It's a useless try that is not necessary here